internal/cloudapi/v2/v2.go

```diff
@@ -165,6 +165,7 @@ func (h *apiHandlers) PostCompose(ctx echo.Context) error {
 	}
 	imageType, err := arch.GetImageType(imageTypeFromApiImageType(ir.ImageType))
 	if err != nil {
+		println("image type from api image")
```

Comment:
`ctx.Logger()` or `logrus.` if you want log statement; though I'm guessing this is leftover debugging.

Also, there's a `println` in go?!